internal/k6runner/k6runner.go

```diff
@@ -99,6 +99,13 @@ func (r Processor) Run(ctx context.Context, registry *prometheus.Registry, logge
 		return false, err
 	}
 
+	if err := k6LogsToLogger(result.Logs, logger); err != nil {
```

Comment:
nit: We could add a clarification comment stating something such as _"Read logs first so they are returned from prober even if metrics can't be parsed_. In any case the test in place will fail if this is moved in the future.

Reply:
Makes sense!